distributed/protocol/tests/test_protocol.py

```diff
@@ -412,3 +414,22 @@ def test_sizeof_serialize(Wrapper, Wrapped):
     assert size <= sizeof(ser_obj) < size * 1.05
     serialized = Wrapped(*serialize(ser_obj))
     assert size <= sizeof(serialized) < size * 1.05
+
+
+@pytest.mark.skipif(WINDOWS, reason="On windows this is triggering a stackoverflow")
```

Comment:
Does this mean that with this change, deeply nested graphs may become unserializable on Windows?